HW7/src/com/jvprogram/singeltonreflecs/AppSingleTon.java

```diff
@@ -0,0 +1,21 @@
+package com.jv_program.singeltonreflecs;
+
+public class AppSingleTon {
+
+    private static volatile AppSingleTon myObj;
+
+    public static AppSingleTon getInstance () {
+        AppSingleTon localInst = myObj;
+
+        if (localInst == null) {
```

Comment:
Good job.
Please also read https://habr.com/ru/post/129494/

Reply:
![image](https://user-images.githubusercontent.com/20267148/54498492-e9a94600-4910-11e9-9d9e-2e29eaee7dfa.png)
Please use this example